examples/calendar/backend/models/time.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Model } from "sequelize";
+
+export class Time extends Model {
+  public id!: number; // Note that the `null assertion` `!` is required in strict mode.
+  public timeStart!: number;
+  public timeEnd!: number;
+  public userId!: string; // for nullable fields
```

Comment:
is this nullable?

Reply:
The code is updated. `!` means non nullable, `?` means nullable.